tests/helpers/server.ts

```diff
@@ -14,7 +14,7 @@ function getGraphqlTestContext() {
   return {
     async before() {
       const app = createApp();
-      serverInstance = await startServer(app, PORT);
+      serverInstance = await startServer(app);
       return new GraphQLClient(`http://0.0.0.0:${PORT}/graphql`);
```

Comment:
You no longer use provided `PORT` to start server, but still use it to connect? Why?

Reply:
adjusted to use the de-facto port.